packages/storybook/stories/INTERNAL/AttachmentCard/AttachmentCard.stories.tsx

```diff
@@ -1,21 +1,27 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { Icon } from '@fluentui/react';
+import { Icon, useTheme } from '@fluentui/react';
 import { _AttachmentCard } from '@internal/react-components';
 import { Meta } from '@storybook/react/types-6-0';
 import React from 'react';
+import { FluentV9ThemeProvider } from '../../../../react-components/src/theming/FluentV9ThemeProvider';
 import { COMPONENT_FOLDER_PREFIX } from '../../constants';
 import { hiddenControl } from '../../controlsUtils';
 
 const AttachmentCardStory = (args): JSX.Element => {
+  const theme = useTheme();
   return (
-    <_AttachmentCard
-      fileName={args.fileName}
-      fileExtension={args.fileExtension}
-      actionIcon={<Icon iconName={args.actionIconName} />}
-      progress={args.progress}
-    />
+    <FluentV9ThemeProvider v8Theme={theme}>
```

Comment:
As we don't use fluent v9 theme publicly right now, could you 
- add a comment that `_AttachmentCard` component depends on Fluent v9 theme **or**
- create a new component, use `FluentV9ThemeProvider` there and use that component everywhere instead of `_AttachmentCard`

Reply:
I wonder if this would work since message thread contains attachment card and message thread has already been wrapped by a V9 wrapper. 


Or we can have components called attachmentCardV9 and attachmentCard, and in message thread it will use attachmentCard 

Is this what you mean?

Reply:
The `_AttachmentCard` component is not exposed publicly, only used internally by the `MessageThread` component. 
The internal storybook will need to use this, so I think this is OK for now.

It would be a problem, if this were an exported component.

Reply:
A comment was added for the internal component